tests/test_rechunk.py

```diff
@@ -62,12 +62,9 @@ def test_rechunk_dataset(
 ):
     if target_store.startswith("mapper"):
         target_store = fsspec.get_mapper(str(tmp_path) + target_store)
-    else:
-        target_store = str(tmp_path / target_store)
-
-    if temp_store.startswith("mapper"):
         temp_store = fsspec.get_mapper(str(tmp_path) + temp_store)
     else:
+        target_store = str(tmp_path / target_store)
```

Comment:
This commit is a left-over from #64

Reply:
I'm not sure I understand your comment. Is it a merge issue?

Reply:
Nope. I was too slow to send it in that PR. Sorry :grimacing:

If you want I can remove it from this PR and do a new one.

Reply:
No it's fine here.